src/player-actions/violence/actions.ts

```diff
@@ -27,11 +29,21 @@ export const alterMomentum = (momentum: number): AppActions => ({
   momentum
 });
 
```

Comment:
(when not using the toolkit), Generally, I'd add the "actionTypes" within this file, so it's all in one place

e.g. `const SELECT_PLAYER_COMBAT_ACTOIN = 'SELECT_PLAYER_COMBAT_ACTION'` to keep code closer to where it's most relevant